app/views/dashboard/_hosts_expired_token_widget_host_list.html.erb

```diff
@@ -0,0 +1,18 @@
+<table class="table table-striped table-fixed">
+  <thead>
+    <th width="30%"><%= _('Host') %></th>
+    <th><%= _('Owner') %></th>
+    <th><%= _('Created') %></th>
+    <th><%= _('Token Expired') %></th>
+  </thead>
+  <tbody>
+    <% hosts.each do |host| %>
+      <tr>
+        <td class="ellipsis"><%= icon_text('error-circle-o', '', kind: 'pficon') %> <%= link_to host.name, host_path(host) %></td>
+        <td class="hidden-tablet hidden-xs ellipsis"><%= host.owner %></td>
+        <td class="hidden-tablet hidden-xs ellipsis"><%= date_time_relative_value(host.created_at) %></td>
```

Comment:
Would getting a value for "Built" be better then created_at? Just created_at wouldn't be relevant to hosts that have been rebuilt.

You could use Setting[:token_duration] and `host.token.expires` to work it out. Obviously that would be incorrect if someone changes Setting[:token_duration] though :(